src/funding/funding.js

```diff
@@ -10,9 +10,9 @@ import type { OnShippingChange } from '../ui/buttons/props';
 
 import { getFundingConfig } from './config';
 
-export function isFundingEligible(source : $Values<typeof FUNDING>, { layout, platform, fundingSource, fundingEligibility, components, onShippingChange, flow } :
+export function isFundingEligible(source : $Values<typeof FUNDING>, { layout, platform, fundingSource, fundingEligibility, components, onShippingChange, flow, enableFunding } :
```

Comment:
There are a lot of points in this PR where we're passing through `enableFunding` but not using it anywhere. Can we clear those up?